### ISSUE TITLE
Add missing migrations

### ISSUE THREAD

**Issue by ggwpez** [paritytech/polkadot#7187]
The following pallets have wrong on-chain version as reported by https://github.com/paritytech/substrate/pull/13417 cc @bkchr.    
CI output from here shows them https://github.com/paritytech/polkadot/pull/7186.
First we have to confirming manually that they are wrong, then we can include the correct ones.

- [x] Polkadot [#7199](https://github.com/paritytech/polkadot/pull/7199)
  - [x] `Historical`
  - [x] `Referenda`
  - [x] `ParasDispute`
  - [x] `Crowdloan`
- [x] Kusama [#7199](https://github.com/paritytech/polkadot/pull/7199)
  - [x] `Historical`
  - [x] `NisCounterpartBalances`
  - [x] `ParasDispute`
  - [x] `Crowdloan`
- [ ] Westend [#7199](https://github.com/paritytech/polkadot/pull/7199), https://github.com/paritytech/substrate/pull/14174, https://github.com/paritytech/polkadot/pull/7251
  - [ ] `FastUnstake`
  - [ ] `ParasDisputes`
  - [ ] `Crowdloan`
- [ ] Rococo (10 total) https://github.com/paritytech/polkadot/issues/6657

**Comment by liamaharon** [comment]
Confirmed https://github.com/paritytech/polkadot/pull/7199 fixes all Polkadot & Kusama issues.

also confirmed the issues with Westend and Rococo, I'll jump on those as relatively high priority since they're causing `on_runtime_upgrade` to fail when run against those chains.

**Comment by liamaharon** [comment]
I need to get more hands on experience with migrations, happy to take on fixing #7186

**Comment by ggwpez** [comment]
Yes please. I think i accidentally removed the changes that were supposed to fix Westend.  
You can re-purpose it to just fix Westend. It seems like we are not going to skip .42 on Polkadot, so the backports are not needed.

**Comment by liamaharon** [comment]
Fixed in https://github.com/paritytech/polkadot/pull/7251 and https://github.com/paritytech/substrate/pull/14174
